### ISSUE TITLE
Webservice checks not executed

### ISSUE THREAD

**Issue by marcellangmaier** [Cacti/plugin_webseer#39]
**Describe the bug**
On a frehsly installed cacti (1.2.8) installation with enabled Webseer plugin (3.0) the webservice checks are not running.

**Expected behavior**
After creating a new check and enabling it it should be run.
I have searched the cacti logfile but can not see anything about webseer there.

**Screenshots**
![grafik](https://user-images.githubusercontent.com/26068556/72518287-4b01b480-3855-11ea-80ab-84714126f8f5.png)

**Plugin (please complete the following information):**
 - Version: 3.0
 - Source: github

**Additional context**

**Comment by netniV** [comment]
Have you enabled the plugin as well as installed it?

**Comment by Alexisclv** [comment]
Hi, I have the exact same things here. 
The plugin is enabled in the plugin list like you said.

![image](https://user-images.githubusercontent.com/45903953/89542155-94205280-d7ff-11ea-8063-0d869bfa6a24.png)

**Comment by tacerus** [comment]
I face the same issue.

![image](https://user-images.githubusercontent.com/5121046/180883420-ccecf6d6-0818-4fab-bc00-ff3524a21035.png)
![image](https://user-images.githubusercontent.com/5121046/180883443-d3059505-2c41-46c3-bc65-27bf9556d7cc.png)
![image](https://user-images.githubusercontent.com/5121046/180883714-501c939d-ab07-42eb-ac51-df2c4f1b9fd8.png)

The "History" pages show "No Events in History". There was an initial error caused by the missing `php-curl` library, however I have resolved that and restarted the webserver. I tried to re-add the server and re-enable the plugin - there are no more error messages now, but the checks are still not executed.

Any ideas on what I could try or check?
